services-toolkit/explanation/four-levels-of-service-consumption.hbs.md

```diff
@@ -0,0 +1,69 @@
+# Four Levels of Service Consumption in Tanzu Application Platform
+
+Over time, as Tanzu Application Platform (referred to as "TAP" from hereon in) has evolved and matured, so has the way in which services can be offered and consumed on the platform. In this document we chart the progress of this evolution in terms of four "levels" of service consumption.
```

Comment:
*thought:* This doc, puts a lot of emphasis on the "evolutionary" and "temporal" aspects of these four levels as they were introduced in the different TAP releases. While that might be interesting to the reader, I wonder if it leaves them with the impression that level 1 was replaced by level 2 which was then replaced by level 3, and so on. You mention in the summary that each level has its pros and cons and different organizational scenarios might be served better by different levels.  I wonder if we should highlight this no-one-size-fits-all nature a bit more throughout the earlier parts of the doc. Doesn't have to be massive changes to what you have right now, just mention that even though we introduced level n, levels 0 to n-1 didn't become obsolete. In fact, in most cases level n builds on top of level n-1, and merely represents a higher-level, slightly more opinionated abstraction.